content/docs/projects/trust-manager.md

```diff
@@ -56,79 +70,299 @@ spec:
         linkerd.io/inject: "enabled"
 ```
 
-Bundle currently supports the source types `configMap`, `secret` and `inLine`,
-and target type `configMap`.
+`Bundle` resources currently support several source types:
+
+- `configMap` - a `ConfigMap` resource in the trust-manager namespace
+- `secret` - a `Secret` resource in the trust-manager namespace
+- `inLine` - a manually specified string containing at least one certificate
+- `useDefaultCAs` - usually, a bundle of publicly trusted certificates
```

Comment:
can we be a bit more specific here and explain where more info can be found?

Reply:
Yeah that'd be an improvement; I think realistically, we should have API reference docs for trust-manager like we have for cert-manager because honestly at this point I don't really think we have a good place where "more info can be found" :joy:

I'll take a quick stab at generating API docs - maybe it's achievable for this PR.